src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -80,6 +80,11 @@ private string GetIncludePath(RuleSet parent)
             List<string> found = new List<string>();
             string expandedPath = PortableShim.Environment.ExpandEnvironmentVariables(_includePath);
 
+            if (PathUtilities.IsUnixLikePlatform)
+            {
+                expandedPath = expandedPath.Replace('\\', PortableShim.Path.DirectorySeparatorChar);
```

Comment:
Feel like this is an issue that MSBuild needed to tackle at a much bigger level.  Have we reached out to them and seen what conclusion they came to?